src/ngraph_builder.cc

```diff
@@ -2309,6 +2309,34 @@ static Status TranslateSigmoidOp(
   return Status::OK();
 }
 
+static Status TranslateSizeOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+
+  DataType dtype;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "out_type", &dtype));
+
+  // Size has an attribute to specify output, int32 or int64
+  ng::element::Type type;
+  TF_RETURN_IF_ERROR(TFDataTypeToNGraphElementType(dtype, &type));
+
+  auto ng_input_shape = ng_input->get_shape();
+
+  int32 result = 1;
+  for (auto dim : ng_input_shape) {
+    result *= dim;
+  }
```

Comment:
We could use int64/long here.
tensor sizes might be very big, eg: 128*64*64*512 etc

Reply:
Yes, I will make this change now.